docs/sources/features/explore/index.md

```diff
@@ -51,6 +51,45 @@ In split view, timepickers for both panels can be linked (if you change one, the
 
 You can close the newly created query by clicking on the Close Split button.
 
+## Query history
+
+> BETA: Query history is a beta feature.
+
+Query history is a list of queries that you have used in Explore. The history is local to your browser and is not shared with others. To open and interact with your history, click the "Query history" button in Explore.
+
+#### View query history
+
+Query history lets you view the history of your querying. For each individual query, you can:
+
+- Replace Explore query for selected query by clicking on the query card
+- Create and/or edit comment
+- Copy query to clipboard
+- Copy URL link with the query to clipboard
+- Star query
+
+#### Manage favorite queries
+
+All queries that are starred are displayed in "Starred" tab as well. This allows you to reuse queries that you run often without typing them from scratch.
+
+#### Sort query history
+
+By default query history shows you the most recent queries. This can be changed and history can be sorted by date or by data source name in ascending or descending order. It is possible to sort queries in "Query history" and "Starred" tab. If you are in split mode, chosen sorting mode is applied only to currently active panel.
```

Comment:
This is a good example of telling people what is possible to do, but for tasks you need to tell them how to do it.

1. Click the **Sort** field.
2. Select one of the following options:
   * **Option** - Define option.

> **Note:** Sorting only applies to the currently active panel, and it does not persist beyond your current Grafana session.

Reply:
Also, be aware of passive constructions like "This can be changed" and "It is possible to." Try to write in the active voice, addressing the second person imperative. "You do this."

Example: You can sort history by date or by data source name. (Followed by instructions for how to do it.)